test/service-stubs/orchestrator.js

```diff
@@ -6,7 +6,7 @@ const router = require('express').Router();
 const logger = require('app/components/logger')('Init');
 const app = express();
 const orchestratorServicePort = config.services.orchestrator.port;
-const orchestratorServicePath = config.services.orchestrator.path;
+const orchestratorServicePath = config.services.orchestrator.paths.submissions;
```

Comment:
why has this changed when the ticket is specifically for removing persistence service information/code

Reply:
Because it was wrong :)

Reply:
```
  orchestrator:
    url: 'http://localhost:8888'
    paths:
      forms: '/forms/case/{ccdCaseId}'
      create: '/forms/newcase/'
      submissions: '/forms/{ccdCaseId}/submissions'
      payments: '/forms/{ccdCaseId}/payments'
      payment_updates: '/payment-updates'
      payment_submissions: '/forms/{ccdCaseId}/payment-submissions'
      fees: '/forms/{ccdCaseId}/fees'
      validations: '/forms/{ccdCaseId}/validations'
      applications: '/forms/cases'
      declarationStatuses: '/invites/{ccdCaseId}'
```